mediator/med_phases_aofluxes_mod.F90

```diff
@@ -1151,6 +1174,43 @@ subroutine med_aofluxes_update(gcomp, aoflux_in, aoflux_out, rc)
        if (chkerr(rc,__LINE__,u_FILE_u)) return
     end if
 
+    ! compute DMS fluxes to atm and ocn
+    if (is_local%wrap%aoflux_grid == 'ogrid') then
+       if (compute_dms_flux) then
+
+          call ESMF_FieldBundleGet(is_local%wrap%FBImp(compocn,compocn), 'So_dms', field=field_src, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+          call ESMF_FieldGet(field_src, farrayptr=odms, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+
+          call ESMF_FieldBundleGet(is_local%wrap%FBImp(compocn,compocn), 'So_t', field=field_src, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+          call ESMF_FieldGet(field_src, farrayptr=sst, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+
+          call ESMF_FieldBundleGet(is_local%wrap%FBMed_aoflux_o, 'So_u10', field=field_src, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+          call ESMF_FieldGet(field_src, farrayptr=u10m, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+
+          call ESMF_FieldBundleGet(is_local%wrap%FBMed_aoflux_o, 'Faox_dms', field=field_src, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+          call ESMF_FieldGet(field_src, farrayptr=flux_dms, rc=rc)
+          if (chkerr(rc,__LINE__,u_FILE_u)) return
+
+          ! flux_dms from ocean is in kg/m2/s
+          ! flux is downwards positive - therefore negative values to the atmosphere  (this is the opposite of what is there in BLOM)
+          do n = 1,size(sst)
+             sst_c = sst(n) - 273.15_r8
+             sst_c = min(40.,max(-3.,sst_c))
+             scdms = 2855.7+  (-177.63 + (6.0438 + (-0.11645 + 0.00094743*sst_c)*sst_c)*sst_c)*sst_c
+             kwdms = Xconvxa * u10m(n)**2 * (660./scdms)**0.5 
+             flux_dms(n) = -62.13 *kwdms * odms(n)
```

Comment:
So many naked constants! Can they be documented somehow (perhaps with a paper reference)?

Reply:
I've added a pointer to the BLOM documentation and source code.